tests/test-scheduled-downtimes.c

```diff
@@ -266,7 +262,6 @@ START_TEST(host_multiple_fixed_scheduled_downtimes_one_cancelled_one_stopped)
 
```

Comment:
Do we need the one just above this line as well? Given that we call `unschedule_downtime` before accessing `is_in_effect`.

Reply:
you are right, missed that one.